src/components/Documents/DocumentTable.jsx

```diff
@@ -32,6 +37,7 @@ const DocumentTable = () => {
     'Upload Date',
     'Expiration Date',
     'Preview File',
+    'Permissions',
```

Comment:
Just a thought, and something we can revisit in a separate PR:

'Permissions' to me sounds a bit stiff and technical. What about naming this feature "sharing?" So instead of "setting permissions," you "share" or "unshare" a document.

Reply:
Align worked, thanks. Sharing sounds good to me, I can edit stuff to say that if you'd like. Or we can do it in another PR like you said.

Reply:
I feel that's part of this PR. 

- The column title in the document table is added by this PR
- The modal to "share" the file or folder is added by this PR

So changing either of those should be part of this PR, in my opinion.

Reply:
I like this idea. Think we could use "share/unshare" for permission setting.